### DISCUSSION TITLE
Run wefwef in Fly.io

### DISCUSSION THREAD

**Discussion by altendorfme** [aeharding/voyager#97] [Show and tell]
Wefwef deploy alternative on Fly.io freetier

If this is your first time on Fly.io, create an account and [install ](https://fly.io/docs/hands-on/install-flyctl/)
After installation, run `fly auth user`

Create the file `fly.toml`
<pre>
app = "<b>wefwef</b>"
kill_signal = "SIGINT"
kill_timeout = "5s"

[experimental]
  auto_rollback = true

[build]
  image = "ghcr.io/aeharding/wefwef:0.8.0"

[env]
  PORT = "5314"

[[services]]
  protocol = "tcp"
  internal_port = 5314
  processes = ["app"]

  [[services.ports]]
    port = 80
    handlers = ["http"]
    force_https = true

  [[services.ports]]
    port = 443
    handlers = ["tls", "http"]
  [services.concurrency]
    type = "connections"
    hard_limit = 25
    soft_limit = 20

  [[services.tcp_checks]]
    interval = "15s"
    timeout = "2s"
    grace_period = "1s"
    restart_limit = 0
</pre>

Then run `fly launch`

<pre>
? Would you like to copy its configuration to the new app? <b>Yes</b>
? Choose an app name (leaving blank will default to 'wefwef') <b>xxx</b>
? Would you like to set up an PostgresSQL database now? <b>No</b>
? Would you like to set up an Upstash Redis database now? <b>No</b>
? Would you like to deploy now? <b>Yes</b>
Visit your newly deployed app at <b>https://xxx.fly.dev/</b>
</pre>

**Comment by aeharding** [comment]
Converting to a discussion. Thanks for sharing! 💙
